### PR TITLE
GH-145: Added Next Map to Maps module.

### PR DESCRIPTION
**Pull request by AtomicLiquid**

### CODE REVIEW

File: src/Modules/NextMapModule/Interfaces/INextMapService.cs [hunk 1]
```diff
@@ -0,0 +1,8 @@
+﻿using EvoSC.Common.Interfaces.Models;
+
+namespace EvoSC.Modules.Official.NextMapModule.Interfaces;
+
+public interface INextMapService
+{
+    public Task<IMap> GetNextMapAsync();
```
Comment (snixtho) [inline src/Modules/NextMapModule/Interfaces/INextMapService.cs:7]: Missing doc comment

File: src/Modules/NextMapModule/NextMapModule.cs [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+[Module(IsInternal = true)]
+public class NextMapModule : EvoScModule, IToggleable
+{
+
+    public NextMapModule()
+    {
+    }
+
+    public Task EnableAsync() => Task.CompletedTask;
```
Comment (snixtho) [inline src/Modules/NextMapModule/NextMapModule.cs:14]: Remove unuses toggle methods.

File: src/Modules/NextMapModule/Templates/NextMap.mt [hunk 3]
```diff
@@ -0,0 +1,18 @@
+﻿<component>
+    <import component="EvoSC.Theme" as="Theme" />
+    <property name="mapName" type="string" />
+    <property name="author" type="string" />
+    
+    <template>
+        <Theme />
```
Comment (snixtho) [inline src/Modules/NextMapModule/Templates/NextMap.mt:7]: Are you even using the theme styles?

**Review by snixtho — APPROVED** [review]

File: src/EvoSC.Testing/Mocking.cs [hunk 4]
```diff
@@ -179,7 +179,7 @@ public static Mock<IContextService> NewContextServiceMock(IControllerContext con
     /// </summary>
     /// <param name="contextService">The context service to use for this localization manager.</param>
     /// <returns></returns>
-    public static Locale NewLocaleMock(IContextService contextService)
+    public static Locale NewLocaleMock(IContextService? contextService)
```
[the remark this answers is not in the document]
Comment (AtomicLiquid) [reply src/EvoSC.Testing/Mocking.cs:182]: Fat fingers, not intentional.

File: src/Modules/NextMapModule/Interfaces/INextMapService.cs (hunk 1 above)
Comment (AtomicLiquid) [reply src/Modules/NextMapModule/Interfaces/INextMapService.cs:7]: Will fix!

File: src/Modules/NextMapModule/Templates/NextMap.mt (hunk 3 above)
Comment (AtomicLiquid) [reply src/Modules/NextMapModule/Templates/NextMap.mt:7]: No, not at the moment, but the intention is to configure this component to use the theme later.

**Pull request merged**